runtime/modules/java/module/src/main/java/io/atlasmap/java/module/JavaValidationService.java

```diff
@@ -86,11 +86,6 @@ public void init() {
         versionMap.put("1.1", 45);
     }
 
-    public void destroy() {
```

Comment:
why removing destroy()?

Reply:
It has no references

Reply:
I would think this kind of lifecycle methods are added for future use...

Reply:
Its not look like a framework method as this method is not in the parent class BaseModuleValidationService.java. I will add it back to make sure we dont get any issues